pkg/operator/ceph/object/config.go

```diff
@@ -54,8 +54,15 @@ func rgwFrontend(v cephver.CephVersion) string {
 	return rgwFrontendName
 }
 
-// TODO: these should be set in the mon's central kv store for mimic+
 func (c *clusterConfig) defaultFlags() []string {
```

Comment:
Can this method be removed now?

Reply:
I missed that we were not supporting Luminous anymore so yes I'll remove it.